TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/AbstractNodeData.php

```diff
@@ -177,7 +178,7 @@ public function setProperty($propertyName, $value)
                 return;
             }
 
-            $this->properties[$propertyName] = $value;
+            $this->properties[$propertyName] = is_string($value) ? Utility::removeControlCharactersFrom($value) : $value;
```

Comment:
I'm not so happy doing it at this level of the CR. A string is a string and the property mapper, content cache and import / export have to deal with encoding properly.

I know this is currently a problem and it occured to us / me in some situations, too. But I'm all in favor of doing it at the level where we want to have some knowledge about the type of string that is stored.

This way we'll have a hard time changing this behavior because it's done at the lowest level of the CR.